queries/beancount/folds.scm

```diff
@@ -1,3 +1,4 @@
 [
     (transaction)
+    (header)
```

Comment:
Looks like this node is called heading, not header

https://github.com/polarmutex/tree-sitter-beancount/blob/79ae7c1f2654a2a6936b0f37bf754e5ff59c9186/grammar.js#L646-L649

Reply:
Yeah you are right. Fixed